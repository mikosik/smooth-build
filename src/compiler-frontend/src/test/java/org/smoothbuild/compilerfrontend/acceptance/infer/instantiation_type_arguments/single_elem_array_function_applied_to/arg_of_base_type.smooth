[A] singleElement(A a) = [a];
myValue = singleElement("abc");
