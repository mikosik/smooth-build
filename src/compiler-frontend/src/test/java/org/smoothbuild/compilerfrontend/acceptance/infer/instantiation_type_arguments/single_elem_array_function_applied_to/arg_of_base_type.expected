[A] singleElement<A>(A a)
  = [a];
[String] myValue<>
  = singleElement<String>("abc");
